lib/DownloadHandler.py

```diff
@@ -305,6 +305,7 @@ def store_file(self, response_content, content_type, url):
         return wd, filename
 
     def download_site(self, url):
+        from lib.Config import Configuration as conf
```

Comment:
I would use a class variable; something like self.config = Configuration(). And move the import statement to the top of the file

Reply:
Fixed in 2e859a3